### ISSUE TITLE
Actions keyboard navigation regression

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/nextcloud-vue#3053]
I noticed the keyboard nav is broken, when opening, the first item should be focused, it is not the case anymore.

1. Open the menu
2. Use arrow down to go to the second item
3. See it doesn't work

You have to tab once to get into the first item THEN you can use arrows keys again

cc @raimund-schluessler

**Comment by raimund-schluessler** [comment]
Likely a regression due to https://github.com/nextcloud/nextcloud-vue/pull/2911.

**Comment by raimund-schluessler** [comment]
Fixed with floating-vue@1.0.0-beta.19 and https://github.com/nextcloud/nextcloud-vue/pull/3504.
